config.default

```diff
@@ -286,11 +286,7 @@ movingAveragePoints = 12
 # compare to observations?
 compareWithObservations = True
 # list of region indices to plot from the region list in [regions] below
-regionIndicesToPlot = [6]
-# Number of points over which to compute moving average (e.g., for monthly
-# output, movingAveragePoints=12 corresponds to a 12-month moving average
-# window)
-movingAveragePoints = 12
+regionIndex = 5
```

Comment:
@vanroekel: I would personally change the section name to something like `Nino34Index`, since you are adding spectra.
To echo @pwolfram comment: don't you need the movingAveragePoints, only different from 12 (something like a 5-month average, if I remember correctly)? If you would like to change the `regionIndex` name, I would probably do it for all scripts consistently.